src/Assets/shadows.sass

```
@@ -0,0 +1,2 @@
+@mixin default-shadow()
+    box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px
```

```suggestion
    box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px
    
```
Add newline at the end of file